.release/ami_install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+
+DIRECTORY=`dirname $0`
+source $DIRECTORY/bootstrap_steps.sh
+
+echo "LOADED UP BOOTSTRAP STEPS"
+
+install_wallaroo() {
+  PYTHON_PATH=$(python -c "import site; print(site.getsitepackages()[0])")
+  git checkout "$WALLAROO_VERSION"
+  sudo cp machida/wallaroo.py "$PYTHON_PATH"
+  (make build-machida && sudo sudo cp machida/build/machida /usr/bin)
+  (cd utils/data_receiver && make && sudo cp ./data_receiver /usr/bin)
+  (cd giles/sender && make && sudo cp ./sender /usr/bin/giles_sender)
+  (cd giles/receiver && make && sudo cp ./receiver /usr/bin/giles_receiver)
+  echo "------ Wallaroo & tools are set up"
+}
+
+install_ponyc
+install_ponyc
```

Comment:
This is a duplicate of line 21.

Also, are functions such as `install_go` (and a few others that appear in `bootstrap_steps.sh`) intentionally omitted here?